client/src/common/constants/variables.js

```diff
@@ -1,6 +1,7 @@
 export const asyncPostfixes = ['FAILURE', 'SUCCESS'];
 export const COMPANY_NAME = 'Sandstream Development Sp. z o.o.';
 export const COMPANY_PAGE_URL = 'https://sandstream.pl/';
+export const COMPANY_WEBSITE = 'www.sandstream.pl/';
```

Comment:
can't we just use this one from line 3?

Reply:
maybe `COMPANY_NAME = 'Sandstream Development Sp. z o.o.';` would be better. Like in app's footer?